webform_civicrm.module

```diff
@@ -1042,3 +1042,20 @@ function webform_civicrm_civicrm_pre($op, $objectName, $id, &$params) {
     }
   }
 }
+
+/**
+ * Implements hook_civicrm_alterPaymentProcessorParams().
+ *
+ * Legacy handling for paypal.
+ * We use it to override the return url so that the user gets redirected to the right place from paypal.
+ *
+ * Remove when dropping support for CiviCRM 4.6 and below.
```

Comment:
CiviCRM 4.6?

Reply:
I think it comes from this commit https://github.com/colemanw/webform_civicrm/pull/244/files#diff-3eeb20e664e204aefaf7a43419a2c9b0afb95a877e37f140c21d024808be057fL573. Deletions on this commit caused this issue as I noticed here https://www.drupal.org/project/webform_civicrm/issues/3111975#comment-13455427

Reply:
@francescbassas We need to update the comment before this PR can be merged - the comment is confusing as it is. If it makes sense just remove the `Remove when dropping support for CiviCRM 4.6 and below.` line.